events.markdown

```diff
@@ -2,21 +2,15 @@
 layout: page
 title: Events
 ---
-### Upcoming Events
+<div id="event-list">
 <br/>
-{%- assign sorted = (site.events | where_exp: "item", "item.date >= site.time" -%}
-{%- for event in sorted -%}
-   [**{{event.title}}**]({{ event.url | relative_url }})
-  {{event.content}}
-{%- endfor -%}
-
+{%- for event in site.events -%}
+<div class="event-link" date="{{ event.date }}">
+<a  href="{{ event.url | relative_url }}" >{{event.title}}</a>
 <br/>
-<hr/>
+{{event.summary}}
 <br/>
-### Past Events
 <br/>
-{%- assign sorted = (site.events | where_exp: "item", "item.date < site.time" -%}
```

Comment:
Why did you need to give on this and move the logic into JS?

Reply:
As I uderstand it, once the site is built, the list is fixed and since these rely on the current date, we need it to be more dynamic